tools/maven-bundle-plugin/src/main/resources/META-INF/m2e/lifecycle-mapping-metadata.xml

```diff
@@ -0,0 +1,40 @@
+<!-- 
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+ 
+    http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License. 
+-->
+<lifecycleMappingMetadata>
+  <pluginExecutions>
+
+    <!-- Goals that should be supported in m2e -->
+    <pluginExecution>
+      <pluginExecutionFilter>
+        <goals>
+          <goal>manifest</goal>
```

Comment:
What about goal `bundle`? The goal manifest is by default not executed for bundle packages (only if configured explicitly, which is rather uncommon for packaging `bundle`). So in most of the projects it would be only the goal `bundle` which is executed but never `manifest`.

Reply:
I guess we need both. Goal manifest is used when you are using packaging jar which I strongly recommend, but of course most people still use packaging bundle.